### PR TITLE
Refactor into reusable includes

### PR DESCRIPTION
**Pull request by lhoss**
this PR builds upon https://github.com/AnsibleShipyard/ansible-zookeeper/pull/32

### CODE REVIEW

File: tasks/tarball.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+
+- file: path=/opt/src state=directory
+- file: path={{zookeeper_dir}} state=directory
+
+- name: Download zookeeper version.
+  get_url: url={{zookeeper_url}} dest=/opt/src/zookeeper-{{zookeeper_version}}.tar.gz
+  tags: bootstrap
+
```
Comment (lhoss) [inline tasks/tarball.yml]: Note: I will remove extra line

File: tasks/tarball.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- name: Create zookeeper data folder.
+  file: path={{data_dir}} state=directory recurse=yes owner=zookeeper group=zookeeper
+  tags: bootstrap
+
+- name: Create zookeeper logs folder.
+  file: path={{log_dir}} state=directory recurse=yes owner=zookeeper group=zookeeper
+  tags: bootstrap
+
```
Comment (lhoss) [inline tasks/tarball.yml]: Note: I will remove extra line

File: tasks/tarball.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  notify:
+    - Restart zookeeper
+
+- name: Configure zookeeper
+  template: src=zoo.cfg.j2 dest={{ zookeeper_dir }}/conf/zoo.cfg owner=zookeeper group=zookeeper
+  tags: deploy
+  notify:
+    - Restart zookeeper
+
+
```
Comment (lhoss) [inline tasks/tarball.yml]: Note: I will remove extra line

**Comment by lhoss** [comment]
@ernestas-poskus plz review (I already noted the empty lines I will remove)

**Comment by ernestas-poskus** [comment]
looks good, tested it in kitchen

remove extra lines and I will merge it

:+1:

File: tasks/tarball.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+
```
Comment (ernestas-poskus) [inline tasks/tarball.yml]: extra

**Comment by lhoss** [comment]
review fixes done

**Comment by ernestas-poskus** [comment]
Thank you for contribution

**Pull request merged**